bin/make/frank-plugin.sh

```diff
@@ -95,6 +101,7 @@ xcrun xcodebuild build \
   ARCHS="i386 x86_64" \
   VALID_ARCHS="i386 x86_64" \
   ONLY_ACTIVE_ARCH=NO \
+  EFFECTIVE_PLATFORM_NAME="-iphonesimulator" \
```

Comment:
I don't really have enough context to ask good questions here, so please bear with me- but why is this line necessary ?